cmd/notation/sign.go

```diff
@@ -35,6 +35,28 @@ func signCommand(opts *signOpts) *cobra.Command {
 	command := &cobra.Command{
 		Use:   "sign [reference]",
 		Short: "Signs artifacts",
+		Long: `
+** This command is in preview and under development. **
```

Comment:
Do you want to follow the same format as `oras`?

Also, we should remove the line `This command is in preview and under development.`. This line is there in oras since some commands like `oras pull` and `oras push` are already used in production.

Reply:
Agree. I will remove this line.